ui/site/src/util.js

```diff
@@ -5,23 +5,30 @@ lichess.engineName = 'Stockfish 8';
 lichess.raf = (window.requestAnimationFrame || window.setTimeout).bind(window);
 lichess.requestIdleCallback = (window.requestIdleCallback || window.setTimeout).bind(window);
 lichess.storage = (function() {
-  var withStorage = function(f) {
-    // can throw an exception when storage is full
-    try {
-      return !!window.localStorage ? f(window.localStorage) : null;
-    } catch (e) {}
-  }
+  try {
+    // just accessing localStorage can throw an exception...
+    var storage = window.localStorage;
+  } catch (e) {}
+  var withStorage = storage ? function(f) {
+    // this can throw exceptions as well.
+    try { return f(storage); }
+    catch (e) {}
+  } : function() {};
```

Comment:
I don't see the point of this change. We're still wrapping every access with `try`. The only case where the behaviour changes is when local storage is not available, which is the corner case we don't care about.